RadixWallet/Core/FeaturePrelude/AddressView/AddressView.swift

```diff
@@ -99,7 +99,6 @@ extension AddressView {
 			}
 		}
 		.multilineTextAlignment(.leading)
-		.minimumScaleFactor(0.5)
```

Comment:
Removed, the stake claim NFT id was made very small. Prefer just truncating the text instead.

Reply:
We should probably keep some kind of scale factor, otherwise adressess will never scale. How about 0.8?

Reply:
We do have an option for showing a full address. But in general, the multilineTextAlignment, should be moved inside group, since it only applies for the first case. And you might keep the scale factor there too.

Then you can ditch the Group and make it a @ViewBuilder instead